azure-pipeline-templates/verbose-tests.yml

```diff
@@ -70,6 +76,7 @@ steps:
       NIGHTLY_STO_ACC_KEY: ${{ parameters.account_key }}
       ACCOUNT_ENDPOINT: ${{ parameters.account_endpoint }}
       VERBOSE_LOG: ${{ parameters.verbose_log }}
+      USE_HTTP: false
```

Comment:
most places we are passing it down as "false" only then we can define with some default value and pass it only when true is required.

Reply:
how do you make env as a default?

Reply:
but we are sending everywhere false only right ? then why to send at all